backend/profileTable.sql

```diff
@@ -0,0 +1,14 @@
+-- @block
+CREATE TABLE IF NOT EXISTS profileTable(
+    accountID INT NOT NULL,
```

Comment:
Would it be possible to specify `PRIMARY KEY` on this line instead of separately on line 11? That would eliminate the need for `NOT NULL` here.

Reply:
From what I've tried it doesn't work. Removing NOT NULL works fine if that's good.

Reply:
Huh that's weird. Admittedly I'm not very familiar with SQL schemata, so ideally I'd want someone else more familiar to chime-in. I suppose this is fine though.